starport/templates/typed/genesis.go

```diff
@@ -227,3 +233,17 @@ func PatchGenesisTypeImport(replacer placeholder.Replacer, content string) strin
 
 	return content
 }
+
+// GenesisStateFieldCount returns the count of field in the genesis state proto message to determine the field numbers
+func GenesisStateFieldCount(path string) (int, error) {
+	pkgs, err := protoanalysis.Parse(context.Background(), nil, path)
+	if err != nil {
+		return 0, err
+	}
+	m, err := pkgs[0].MessageByName(ProtoGenesisStateMessage)
```

Comment:
maybe we should add a check for the pkgs length, in case is empty

Reply:
Good catch, I thought `Parse` would return an error but it's actually not the case